### ISSUE TITLE
Batch Normalization | 慕湮

### ISSUE THREAD

**Issue by tycallen** [tycallen/gitment-comments#13]
http://muyaan.com/2018/08/21/Batch-Normalization/
